cmd/minishift/cmd/completion.go

```diff
@@ -31,15 +31,20 @@ const longDescription = `
 	Outputs minishift shell completion for the given shell (bash or zsh)
 
 	This depends on the bash-completion binary.  Example installation instructions:
+
 	OS X:
+
 		$ brew install bash-completion
 		$ source $(brew --prefix)/etc/bash_completion
-		$ minishift completion bash > ~/.minishift-completion  # for bash users
-		$ minishift completion zsh > ~/.minishift-completion  # for zsh users
+		$ minishift completion bash > ~/.minishift-completion	# for bash users
+		$ minishift completion zsh > ~/.minishift-completion	# for zsh users
 		$ source ~/.minishift-completion
-	Fedora:
-		$ dnf install bash-completion
-		$ minishift completion bash > ~/.minishift-completion  # for bash users
+
+	RHEL/Fedora:
+
+		$ yum install bash-completion				# for RHEL
+		$ dnf install bash-completion   			# for Fedora
+		$ minishift completion bash > ~/.minishift-completion 	# for bash users
```

Comment:
The instructions for both `bash` and `zsh` were included for macOS. Is there a reason that only `bash` is included here?

Reply:
@robin-owen yes, because bash is default in Linux.

Reply:
@LalatenduMohanty I'm aware, but may be biased, as I use zsh. ;)

Reply:
@robin-owen I never used zsh, but if you steps which I can add here, we should be able to add the information.

Reply:
@LalatenduMohanty I just verified that the steps used for macOS work properly on Fedora 27 using zsh. Specifically:

```sh
$ minishift completion zsh > ~/.minishift-completion
$ source ~/.minishift-completion
```

Reply:
@robin-owen Added zsh step. Please check.